pyscal/pyscallist.py

```diff
@@ -300,8 +302,9 @@ def interpolate(
         """
 
         if self.pyscaltype != SCALrecommendation:
-            logger.error("Can only interpolate PyscalList of type SCALrecommendation")
-            raise TypeError
+            raise TypeError(
```

Comment:
Why change this here but not here https://github.com/equinor/pyscal/blob/b36adf62664547b9b9fd26948b0f1d068b26532c/pyscal/pyscallist.py#L204 ?

Reply:
good catch.